### PR TITLE
Seven/add store pickup to order response

### PR DESCRIPTION
**Pull request by Georgemanif**

### CODE REVIEW

File: source/localizable/smart_cart/_order_object.html.md.erb [hunk 1]
```diff
@@ -22,6 +22,7 @@ Name         | Type   | Value | Description
 `reject_options` | Object | | [Reject options](#reject-options) (available for orders with state `"open"`)
 `fulfilled_by_skroutz` | Boolean | | Whether the order is fulfilled by skroutz
 `fbs_delivery_note` | String | | Order's delivery note from warehouse for orders that are fulfilled by Skroutz
+`store_pickup` | Boolean | | Whether the order will be pickup up by the user at the store
```
Comment (alexandrosth) [inline source/localizable/smart_cart/_order_object.html.md.erb:25]:
```suggestion
`store_pickup` | Boolean | | Whether the order will be picked up by the user at the store
```

File: resources/responses/skroutz/merchants_ecommerce_orders_demo_store_pickup.yml [hunk 2]
```diff
@@ -0,0 +1,97 @@
+---
+:request:
+  :url: https://api.skroutz.gr/merchants/ecommerce/orders/DEMO-FBS
```
Comment (alexandrosth) [inline resources/responses/skroutz/merchants_ecommerce_orders_demo_store_pickup.yml:3]:
```suggestion
  :url: https://api.skroutz.gr/merchants/ecommerce/orders/DEMO-STORE-PICKUP
```

File: resources/responses/skroutz/merchants_ecommerce_orders_demo_store_pickup.yml [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    X-App: as103_ssl
+    X-Varnish: 989369834
+    Age: 0
+    Via: 1.1 varnish (Varnish/5.0)
+    X-Origin: lb2
+    X-Cache: MISS
+    X-Cached: skroutz miss(0)
+  :body: '{
+  "order": {
+    "code": "DEMO-FBS",
```
Comment (alexandrosth) [inline resources/responses/skroutz/merchants_ecommerce_orders_demo_store_pickup.yml:29]:
```suggestion
    "code": "DEMO-STORE-PICKUP",
```

File: source/localizable/smart_cart/orders_api.html.md.erb [hunk 4]
```diff
@@ -258,6 +258,16 @@ A dispatched order without a courier voucher that will be fulfilled by Skroutz.
 
 <%= render_recording :merchants_ecommerce_orders_demo_fbs_dispatched %>
 
+#### DEMO-STORE-PICKUP
+
+An open order without a courier or courier voucher that will be pickup up by the user at the shop.
```
Comment (alexandrosth) [inline source/localizable/smart_cart/orders_api.html.md.erb:263]:
```suggestion
An open order without a courier or courier voucher that will be picked up by the user at the shop.
```

File: resources/responses/skroutz/merchants_ecommerce_orders_demo_store_pickup.yml [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+        },
+        {
+          "id": 5,
+          "label": "To προϊόν καταργήθηκε",
+          "requires_available_quantity": false
+        }
+      ]
+    }
+  }
+}'
```
Comment (alexandrosth) [inline resources/responses/skroutz/merchants_ecommerce_orders_demo_store_pickup.yml:115]:
```suggestion
}

```

Comment (alexandrosth) [reply resources/responses/skroutz/merchants_ecommerce_orders_demo_store_pickup.yml]: my bad ignore this 
just the new line otherwise it breaks

Comment (Georgemanif) [reply resources/responses/skroutz/merchants_ecommerce_orders_demo_store_pickup.yml]: 183e898ecd7d7447a4340c5aa92ac9383fc88f7e

**Review by alexandrosth — APPROVED** [review]

**Pull request merged**